### PR TITLE
docs: added homepage

### PR DESCRIPTION
**Pull request by rahulyadav139**
Palasio Quiz application is here!. I have created the homepage for it. Kindly review it.

### CODE REVIEW

**Review by Ananyamadhu08 — APPROVED** [review]
looks good to go 👍

**Review by PuneetP16 — APPROVED** [review]
Looks good to merge, just verify rem's less than 1, if not used purposely could change them to px.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+          <li>
+            <a href="/"><btn class="btn primary">Signup</btn></a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <section>
+        <h1 class="brand-name">PALASIO QUIZ</h1>
+        <h1 class="main__intro-text">
```
Comment (muskanbagrecha) [inline index.html]: Are you sure you want both brand name and intro text to be h1?

File: shared.css [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+}
+.theme-toggle__label span:nth-child(1),
+.theme-toggle__label span:nth-child(2) {
+  width: 1.25rem;
+  height: 1.25rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.theme-toggle__label span:last-child {
```
Comment (muskanbagrecha) [inline shared.css:68]: width and height are same for toggle labels. So put them under the common class. DRY - dont repeat yourself.

**Review by muskanbagrecha — CHANGES_REQUESTED** [review]
Great work overall. Loved the UI.

File: index.css [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  padding: 3rem;
+}
+
+.main section .brand-name {
+  color: var(--primary-color-dark);
+  font-size: 2.5rem;
+  font-weight: bold;
+  font-family: 'Quintessential', cursive;
+  letter-spacing: 0.5rem;
+}
```
Comment (manojsarna) [inline index.css:20]: looks good!!

File: index.css [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  gap: 1rem;
+  flex-basis: 50%;
+  padding: 3rem;
+}
+
+.main section .brand-name {
+  color: var(--primary-color-dark);
+  font-size: 2.5rem;
+  font-weight: bold;
+  font-family: 'Quintessential', cursive;
```
Comment (manojsarna) [inline index.css:18]: can use 300-900 font weight values

File: index.css [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+  letter-spacing: 0.5rem;
+}
+
+.main .main__intro-text {
+  font-weight: 500;
+  margin-top: 2rem;
+}
+
+.main .main__intro-subtext {
+  font-weight: 500;
```
Comment (manojsarna) [inline index.css:28]: you have used values here , be consistent with the type of values you are using

File: index.html [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+          Have fun testing your expertise with this general knowledge quiz!
+          Contains thousands of questions from various categories.
+        </h1>
```
Comment (manojsarna) [inline index.html]: tagline is too long!!

File: shared.css [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+  top: 0;
+  right: 0;
+  opacity: 0;
+}
+
+.theme-toggle__label {
+  padding: 5px;
+  background-color: var(--black-color);
+  border-radius: 0.625rem;
+  display: flex;
```
Comment (manojsarna) [inline shared.css:54]: check with the values in rem can you px if less than 10px

File: shared.css [hunk 8]
```diff
@@ -0,0 +1,90 @@
...
+  justify-content: center;
+  align-items: center;
+  position: relative;
+  cursor: pointer;
+}
+.theme-toggle__label span:nth-child(1),
+.theme-toggle__label span:nth-child(2) {
+  width: 1.25rem;
+  height: 1.25rem;
+  display: flex;
```
Comment (manojsarna) [inline shared.css:64]: common class can be formed

**Review by manojsarna — CHANGES_REQUESTED** [review]
Overall Lokks Good!!

See Comments for Changes!!

File: index.html (hunk 1 above)
Comment (rahulyadav139) [reply index.html]: Thank you for pointing that out. I have changed h2 for the second one.

File: shared.css (hunk 2 above)
Comment (rahulyadav139) [reply shared.css:68]: For the readability issue, I repeat that code. For now, I am keeping the same it is.

File: shared.css (hunk 7 above)
Comment (rahulyadav139) [reply shared.css:54]: If I take value in px then it looks bad in the mobile view due to the same border width as in the desktop view. So I changed that in the rem.

File: index.html (hunk 6 above)
Comment (rahulyadav139) [reply index.html]: I don't think this should be an issue.

File: index.css (hunk 5 above)
Comment (rahulyadav139) [reply index.css:28]: I use generally font-weight as a predefined value with bold but sometimes it is not desirable so I change that with specific font-weight

**Review by muskanbagrecha — APPROVED** [review]
Changes approved.

**Pull request merged**